[Due: Apr 4] Docs updates for dedicated projects UI (#591)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/settings/index.mdx

### Project

> The [project](/terraform/cloud-docs/projects) that this workspace belongs to. Changing the workspace's project can change the read and write permissions for the workspace and which users can access it. The user must have admin permissions on both the source and destination project to move a workspace, either through the "Manage all Projects" organization permission, or explicit team admin privileges for both projects. Keep in mind that moving a workspace to another project may affect user visibility for the workspaces in the project. Refer to [Project Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#project-permissions) for more detail on workspace access.

Reviewer: Splitting this up for scan-ability, and did a little sentence restructuring for clarity:
```suggestion
The [project](/terraform/cloud-docs/projects) that this workspace belongs to. Changing the workspace's project can change the read and write permissions for the workspace and which users can access it. 

To move a workspace, you must have the "Manage all Projects" organization permission or explicit team admin privileges on both the source and destination projects. Remember that moving a workspace to another project may affect user visibility for that project's workspaces. Refer to [Project Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#project-permissions) for details on workspace access.
```
